cpp/ps0002/ps0002_sol.cpp

```diff
@@ -0,0 +1,18 @@
+#include <iostream>
+using namespace std;
+int main()
+{
+    int population = 100; // 0.5/0.75
+    char drug_alternatives = 9; // 0.5/0.75
+    char teamMembers = 5; // 0.75/0.75
+    float bakedPrice = 6.5; // 0.5/0.75
+    int birthsNumber = 5000000; // 0.5/0.75
+    char firstLetter = 'M'; // 0.75/0.75
+    double output = 5.54677743; // 0.25/0.75
+    float electricityBill = 100000000; // 0.75/0.75
+    unsigned int adultsAgechildrenAge = 1000000000; // 0.75/0.75
+    // total 5.25/6.75
+
+    return 0;
+}
+// good work 👍 7.5/10
```

Comment:
- [ ] program is in the correct path
- [x] no syntax errors
- [ ] formatted code to be clear and readable
- [ ] commented some phrases for blocks of code